action/recipes/universal.go

```diff
@@ -0,0 +1,99 @@
+// SPDX-License-Identifier: MIT
+
+// Package recipes / universal
+package recipes
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+
+	"dagger.io/dagger"
+	"github.com/9elements/firmware-action/action/container"
+)
+
+// UniversalSpecific is used to store data specific to universal
```

Comment:
I know there comments have just been slightly adapted from the u-root module but they read really weird now. Maybe something like this?
```suggestion
// UniversalSpecific is used to store data specific to the universal command module
```

Reply:
`universal command module` :D sounds like NASA project :D

But yeah, you are right, they sound odd, but I had no idea what to change it to. `universal module` just sounds a but weird.